acapy_agent/ledger/indy_vdr.py

```diff
@@ -321,9 +322,10 @@ async def _submit(
             taa_accept: whether to apply TAA acceptance to the (signed, write) request
             sign_did: override the signing DID
             write_ledger: whether to write the request to the ledger
-
+            profile: The profile context used for the request
```

Comment:
I was thinking a comment here would be good. Some type of note saying why this argument was added and that it shouldn't be needed. The abstract class would be good to have as well but it doesn't look like it has argument descriptions.